packages/core/src/browser/style/notification.css

```diff
@@ -0,0 +1,29 @@
+/********************************************************************************
+ * Copyright (C) 2018 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+.notification-count {
+    align-self: center;
+    background-color: var(--theia-ui-font-color3);
+    border-radius: 20px;
+    color: var(--theia-ui-font-color0);
+    font-size: calc(var(--theia-ui-font-size0) * 0.8);
+    font-weight: 500;
+    height: calc(var(--theia-private-horizontal-tab-height) * 0.7);
+    line-height: calc(var(--theia-private-horizontal-tab-height) * 0.7);
```

Comment:
There is a problem with the line-height. It is interpreted differently in Chrome and Firefox.
This is Chrome now:
<img width="30" alt="screen shot 2018-11-15 at 10 31 30" src="https://user-images.githubusercontent.com/28291421/48543759-763feb00-e8c2-11e8-85f9-ae9865b53cb5.png">

This is FF now:
<img width="31" alt="screen shot 2018-11-15 at 10 30 05" src="https://user-images.githubusercontent.com/28291421/48543769-7d66f900-e8c2-11e8-91dc-5eaa9a86d3ff.png">


So, for Chrome a factor of 0.8 works. For now I don't have an idea how to handle this. Do you?

Reply:
This could work:
Change the HTML of the notifications to
```
<div class="notification-count">
    <span style="align-self: center;">
        5
    </span>
</div>
```
(I used inline-style here, should use a class of course)
 
Remove `line-height` from `notification-count` and add `display: flex` and `justify-content:center` instead.

Reply:
thanks @jbicker for the hint! I'll make the necessary changes